sample/src/main/java/com/microsoft/gctoolkit/sample/Main.java

```diff
@@ -7,20 +7,26 @@
 import com.microsoft.gctoolkit.sample.aggregation.HeapOccupancyAfterCollectionAggregation;
 import com.microsoft.gctoolkit.sample.aggregation.HeapOccupancyAfterCollectionSummary;
 
+import java.nio.file.Files;
 import java.nio.file.Path;
 
 public class Main {
 
     public static void main(String[] args) throws Exception {
+        if (args.length == 0) {
+            throw new IllegalArgumentException("This sample requires a path to a GC log file as an argument.");
+        }
         Main main = new Main();
-        main.execute();
+        main.execute(args[0]);
     }
 
-    public void execute() {
-        GCLogFile logFile = new SingleGCLogFile(Path.of("./gclogs/preunified/cms/defnew/details/defnew.log"));
+    public void execute(String gcLogFilePath) {
+        GCLogFile logFile = new SingleGCLogFile(Path.of(gcLogFilePath));
         GCToolKit gcToolKit = new GCToolKit();
         JavaVirtualMachine machine = gcToolKit.analyze(logFile);
-        HeapOccupancyAfterCollectionAggregation results = machine.getAggregation(HeapOccupancyAfterCollectionSummary.class);
-        System.out.println(results.toString());
+        HeapOccupancyAfterCollectionSummary results = machine.getAggregation(HeapOccupancyAfterCollectionSummary.class);
+        results.get().forEach((gcType, dataSet) ->
+                System.out.println("The XYDataSet for " + gcType + " contains " + dataSet.size() + " items.")
```

Comment:
I think this is more interesting output, but it does not match the output in the introductory blog post.

Reply:
Done in commit 5a8f2ee